opba-protocols/protocol-bpmn-shared/src/test/java/de/adorsys/opba/protocol/bpmnshared/config/flowable/LargeJsonCustomSerializerTest.java

```diff
@@ -0,0 +1,45 @@
+package de.adorsys.opba.protocol.bpmnshared.config.flowable;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LargeJsonCustomSerializerTest {
+    private static LargeJsonCustomSerializer serializer;
+
+    @BeforeAll
+    public static void init() {
+        serializer = new LargeJsonCustomSerializer(null, new ObjectMapper(), Arrays.asList("java.lang.String"), 2048);
+    }
+
+    @Test
+    void serializeLessThanMinValueTest() {
+        String input = generateRandomStringOfLength(2046);
+        assertThat(serializer.isAbleToStore(input)).isFalse();
+    }
+
+    @Test
+    void serializeMoreToMinValueTest() {
+        String input = generateRandomStringOfLength(2047);
+        assertThat(serializer.isAbleToStore(input)).isTrue();
+    }
+
+    private String generateRandomStringOfLength(int length) {
+        int leftLimit = 97;
```

Comment:
Instead of this long function - 
```
Strings.repeat
```
from Guava would solve the issue

Reply:
Replaced with Strings.repeat in all tests